src/context.rs

```diff
@@ -63,8 +63,9 @@ impl Context {
             method,
             params: Some(params.unwrap()),
         };
-        self.lang_srv_tx
-            .send(ServerMessage::Request(Call::MethodCall(call)));
+        if let Some(tx) = &self.lang_srv_tx {
+            tx.send(ServerMessage::Request(Call::MethodCall(call)));
+        }
```

Comment:
Note that this `if let` is suspicious: what should we do if the  `lang_srv_tx` is `None`? Note, however, that the very same problem was here from the start: it could have been the case that the other side of the channel was poisoned, making this message silently lost. This approach just make this subtlety visible in code.